invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/globals/site.variables

```diff
@@ -2,20 +2,43 @@
     Invenio App RDM Site Variables
 ***********************************************/
 
+/*-------------------
+    Brand Colors
+--------------------*/
+
 @secondaryColor: #fb8273;
 @numberedListBackgroundColor: #4899DB;
+@brandColor: @primaryColor;
 
-@fontSizeBase: 14px;
+/*-------------------
+      Site Colors
+--------------------*/
 
-@linkColor: #005b9e;
-@linkUnderline: none;
-@linkHoverColor: darken(saturate(@linkColor, 20), 15, relative);
-@linkHoverUnderline: underline;
+@green: #048622;
+@signupColor: @green;
+@searchButtonColor: #fb8273;
```

Comment:
Shall we define a color instead so that with a name we can immediately understand what color is that? Same above and below?
```suggestion
@searchButtonColor: @<color-name>;
```